paperqa/types.py

```diff
@@ -320,6 +320,9 @@ class DocDetails(Doc):
         " We use None as a sentinel for unset values (like for determining hydration) "
         " So, we use -1 means unknown quality and None means it needs to be hydrated.",
     )
+    is_retracted: bool | None = Field(
```

Comment:
We should add the formatting to `formatted_citation` method so that we can actually see the article is retracted. Maybe but a big ole "RETRACTED ARTICLE: .... This article was retracted"